stix_shifter_modules/secretserver/tests/stix_translation/options.json

```diff
@@ -0,0 +1,64 @@
+{
+  "mapping": {
+    "from_stix_map": {
+      "ipv4-addr": {
+        "fields": {
+          "value": [
+            "IpAddress",
+            "Server"
+          ]
+        }
+      },
+      "url": {
+        "fields": {
+          "value": [
+            "url"
+          ]
+        }
+      },
+      "network-traffic": {
+        "fields": {
+          "src_ref.value": [
+            "IpAddress"
+          ],
+          "dst_ref.value": [
+            "Server"
+          ],
+          "protocols[*]": [
+            "protocolid"
```

Comment:
I'm wondering how this comes back from Secret Server? If it's a numbered ID, it would need to be converted into a protocol and placed in an array. STIX requires the protocols to be written as something like `[tcp, ump]`. The QRadar connector needs to translate numbered IDs into protocols and put them into an array  with the `ToLowercaseArray` transformer.